src/components/ImageSelection/index.js

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react'
+import { View, StyleSheet } from 'react-native'
+import { Overlay } from 'react-native-elements'
+import { Button, Text } from '~/components'
+import ImagePicker from 'react-native-image-crop-picker'
+
+const styles = StyleSheet.create({
+  button: {
+    padding: 20,
+    margin: 5,
+    borderRadius: 2
+  },
+  title: {
+    marginBottom: 10
+  }
+})
```

Comment:
I suggest changing this to `scale/moderateScale`.  DPI units behave differently in different devices according to pixel density. Scaling units are calculated based on screen dimensions, so what you see in the emulator (and with a specific resolution and pixel density) is what you expect to see in other devices too.

You will have to try some values, as 

`padding: 20 !== padding: scale(20)`